Info-Helios.plist

```diff
@@ -7,22 +7,24 @@
 	<key>CFBundleExecutable</key>
 	<string>Helios</string>
 	<key>CFBundleGetInfoString</key>
-	<string>1.5, Freeware</string>
+	<string>1.51, Freeware</string>
```

Comment:
That's quite the version # jump, going from 5 all the way to 51. Let's make it 6 instead.